virtualization-and-containers/material/exercise_vm_containers_text.md

```diff
@@ -0,0 +1,175 @@
+  - Open the issue.
+    - As title choose "[`USERNAME`] Ubuntu Server Installation"
+    - Attach the screenshot ("Attach a file"). By default, the screenshot is saved in the folder of your VM.
+    - If you did **not** assign 2048 MB of memory to the VM, please mention how much memory you assigned to it instead.
+    - Add the `VirtualBox` label to the issue and assign the issue to `jaustar`. Finally, create the issue.
+    - Double-check that your issue looks as expected. You will find an example issue in the GitLab Repository.
+
+Congratulations. You have successfully set up a virtual machine, installed Ubuntu and installed additional software. You are now done with this part of the exercise. If you do not want to play with the VM later nor do you want to do any of the optional assignments, you can poweroff and delete the VM.
+
+### Further Information (VirtualBox)
+
+- If your mouse pointer gets caught, use the **right** `CTRL` (`STRG` on German keyboards) key on your keyboard to release it again.
+- You can find more information on the installation procedure in the [Ubuntu Server documentation](https://ubuntu.com/server/docs).
+- [Ubuntu Server download page](https://ubuntu.com/download/server)
+- [VirtualBox Manual](https://www.virtualbox.org/manual/UserManual.html)
+- If you want to set up SSH forwarding to your VM, you can find instructions, for example, [on the codebot homepage](https://codebots.com/docs/ubuntu-18-04-virtual-machine-setup). The instructions are for Ubuntu 18.04, but it also works for Ubuntu 20.04. Note that most of the steps are preconfigured in current VirtualBox releases, but it does not hurt to check whether the settings are set properly.
+
+
+## Virtual Machines Using Vagrant
+
+In the previous section we have set up a VM manually. This was quite tedious. Therefore, we want to automatize this process now by using [Vagrant](https://www.vagrantup.com/) and setting up our own Vagrant box. We will provision a box based on VirtualBox by using shell scripts and the included provisioning tools of Vagrant.
+
+### Tasks (Vagrant)
+
+- Fork the repository [TODO: "Exercise Virtual Machines"](https://gitlab-sim.informatik.uni-stuttgart.de/sse-test-group/exercise-virtual-machines) from GitLab and create a branch to work on the fork. It will contain a `README.md` and a file called `bootstrap.sh`.
+- We want to start from scratch so initialize a new box using `vagrant init` in inside this repository and add the resulting `Vagrantfile` to Git. Then adapt your `Vagrantfile` to incorporate the following settings:
+  - Your virtual machine must be based on the [`ubuntu/focal64` image](https://www.vagrantup.com/docs/boxes#official-boxes).
+  - The name of your VM must be `USERNAME-ubuntu-server`.
+  - The [box version](https://www.vagrantup.com/docs/boxes/versioning) must be `0.1.0`.
```

Comment:
This does not work. I get an error message that there is no `0.1.0` version of `ubuntu/focal64`.

Reply:
I have to rephrase that. The idea was that you set the version of your own box to 0.1.0.